eng/scripts/TypeSpec-Requirement.ps1

```diff
@@ -20,9 +20,33 @@ function Get-Suppression {
 
   # -NoEnumerate to prevent single-element arrays from being collapsed to a single object
   # -AsHashtable is closer to raw JSON than PSCustomObject
-  $suppressions = npx get-suppressions TypeSpecRequirement $fileInSpecFolder | ConvertFrom-Json -NoEnumerate -AsHashtable
+  $suppressions = npm exec --no -- get-suppressions TypeSpecRequirement $fileInSpecFolder | ConvertFrom-Json -NoEnumerate -AsHashtable
 
-  return $suppressions ? $suppressions[0] : $null;
+  if ($LASTEXITCODE -ne 0) {
+      LogError "Failure running 'npm exec get-suppressions'"
+      LogJobFailure
+      exit 1
+  }
+
+  $suppression = $suppressions ? $suppressions[0] : $null
+
+  if ($suppression) {
+    $path = $suppression["path"]
+
+    # Allowed:    data-plane/Azure.Contoso.WidgetManager/preview/2022-11-01-preview/**/*.json
```

Comment:
This is total nitpick, but I understand these are just two examples, and by looking at these two examples, it is not immediately clear to me what is the exact border where allowed becomes disallowed. Specifically, how much nesting do I need. Which of these are allowed or not?

`Azure.Contoso.WidgetManager/preview/2022-11-01-preview/**/*.json`
`data-plane/Azure.Contoso.WidgetManager/2022-11-01-preview/**/*.json`
`data-plane/Azure.Contoso.WidgetManager/preview/**/*.json`
`data-plane/Azure.Contoso.WidgetManager/**/*.json`

Reply:
Improve comment